Readme.md

```diff
@@ -1,27 +1,104 @@
+
+```xaml
+<dxb:BarButtonItem x:Name="itemCut" Content="Cut" />
+<dxb:BarButtonItem x:Name="itemCopy" Content="Copy" />
+<dxb:BarButtonItem x:Name="itemPaste" Content="Paste" />
+<dxb:BarButtonItem x:Name="itemUndo" Content="Undo" />
+<dxb:BarButtonItem x:Name="itemRedo" Content="Redo" />
+```
+
+### Group Commands in a BarLinkContainerItem
+
+Use the [`BarLinkContainerItem`](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarLinkContainerItem) property to group `Cut`, `Copy`, and `Paste` commands into a single reusable unit. This container holds links to existing [`BarButtonItem`](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarButtonItem) elements:
+
+```xaml
+<dxb:BarLinkContainerItem x:Name="linkContainerItem1" Content="Edit Commands">
+    <dxb:BarLinkContainerItem.ItemLinks>
+        <dxb:BarButtonItemLink BarItemName="itemCut" />
+        <dxb:BarButtonItemLink BarItemName="itemCopy" />
+        <dxb:BarButtonItemLink BarItemName="itemPaste" />
+    </dxb:BarLinkContainerItem.ItemLinks>
+</dxb:BarLinkContainerItem>
+```
+
+### Reuse the Container in a Submenu
+
+You can reuse the container in a [`BarSubItem`](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarSubItem), which acts as a submenu in the UI:
+
+```xaml
+<dxb:BarSubItem Content="Edit" x:Name="subMenu1">
+    <dxb:BarSubItem.ItemLinks>
+        <dxb:BarLinkContainerItemLink BarItemName="linkContainerItem1" />
+    </dxb:BarSubItem.ItemLinks>
+</dxb:BarSubItem>
+```
+
+### Display Items in a Toolbar
+
+The following code example defines a top-level bar and adds both the container and the submenu, along with separators and additional items:
+
+```xaml
+<dxb:Bar x:Name="bar1" Caption="Bar 1">
+    <dxb:Bar.ItemLinks>
+        <dxb:BarLinkContainerItemLink BarItemName="linkContainerItem1" />
+        <dxb:BarItemLinkSeparator />
+        <dxb:BarButtonItemLink BarItemName="itemUndo" />
+        <dxb:BarButtonItemLink BarItemName="itemRedo" />
+        <dxb:BarItemLinkSeparator />
+        <dxb:BarSubItemLink BarItemName="subMenu1" />
+    </dxb:Bar.ItemLinks>
+</dxb:Bar>
+```
+
+## Files to Review
+
+* [Window1.xaml](./CS/BarLinkContainerItem/Window1.xaml) (VB: [Window1.xaml](./VB/BarLinkContainerItem/Window1.xaml))
+* [Window1.xaml.cs](./CS/BarLinkContainerItem/Window1.xaml.cs) (VB: [Window1.xaml.vb](./VB/BarLinkContainerItem/Window1.xaml.vb))
+
+## Documentation
+
+- [BarContainerControl](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarContainerControl)
+- [BarLinkContainerItem](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarLinkContainerItem)
+- [Bars](https://docs.devexpress.com/WPF/6194/controls-and-libraries/ribbon-bars-and-menu/bars?p=netframework)
+
+## More Examples
+
+- [WPF Bars - Use separators to group bar item links](https://github.com/DevExpress-Examples/wpf-bars-visually-separate-bar-items)

```

Comment:
I'll add suggestions to update these topic title links below.
Can you update the actual title of each GitHub page as well?

```suggestion
- [WPF Bars - Use Separators to Group Bar Item Links](https://github.com/DevExpress-Examples/wpf-bars-visually-separate-bar-items)
```